packages/app/src/components/applicationStages/ApplicationStages.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
+import nominationsAPI from '../../utils/API/nominationsAPI';
+import './style.css';
+
+const ApplicationStages = () => {
+  const [activeNomination, setActiveNomination] = useContext(
+    ActiveNominationContext
+  );
+  const [currentStatus, setCurrentStatus] = useState();
+  const status = [
+    'Received',
+    'Awaiting HIPAA',
+    'HIPAA Verified',
+    'Document Review',
+    'Ready for Board Review',
+  ];
+  useEffect(() => {
+    setCurrentStatus(capitalize(activeNomination.status));
+  }, [activeNomination, currentStatus]);
+  function createStatusEl() {
+    const activeStatusIndex = status.indexOf(currentStatus);
+    return status.map((stat, i) => (
+      <>
+        {activeStatusIndex === i ? (
+          <div className="step current">
+            <span>{stat}</span>
+          </div>
+        ) : activeStatusIndex < i ? (
+          <div className="step">
+            <span>{stat}</span>
+          </div>
+        ) : activeStatusIndex > i ? (
+          <div className="step complete">
+            <span className="checkmark">✓</span>
+          </div>
+        ) : null}
+      </>
+    ));
+  }
+  function advanceStage(value) {
+    let index = status.indexOf(value);
+    if (index >= 0 && index < status.length - 1) {
+      let nextItem = status[index + 1];
+      activeNomination.status = nextItem;
+      setCurrentStatus(nextItem);
+      return updateNom(nextItem);
+    }
+  }
+  function updateNom(currentStatus) {
+    nominationsAPI
+      .updateNomination(activeNomination.id, currentStatus)
+      .then((res) => {})
+      .catch((err) => console.log(err));
+  }
+  const capitalize = (s) => {
```

Comment:
This is fine, but fwiw, Create React App includes [Lodash](https://lodash.com/docs/), which has a [capitalize method](https://lodash.com/docs/4.17.15#capitalize)